Surface_mesh_parameterization/doc/Surface_mesh_parameterization/Surface_mesh_parameterization.txt

```diff
@@ -547,6 +547,7 @@ and replaces a wrapper which had a more complicated API.
 Iterative authalic parameterization was added in CGAL 5.2 by Mael Rouxel-Labbé,
 based on a prototype developed by Hardik Jain.
 
+In CGAL 6.0 the fork of OpenNL that was distributed with CGAL was removed.
+
```

Comment:
This suggests that it is still possible to use OpenNL, I thought the usage was completely removed.

Reply:
> I thought the usage was completely removed.

New wording:
> In CGAL 6.0 this package no longer uses the fork of OpenNL that was distributed with CGAL
and hence was removed.

Is my assumption about the removal incorrect >
The new wording does not signal my thoughts either.